physics/cu_gf_driver.meta

```diff
@@ -553,6 +553,22 @@
   type = real
   kind = kind_phys
   intent = in
+[maxupmf]
+  standard_name = maximum_conv_up_draft_mass_flux
```

Comment:
We try not to have abbreviations in standard names. Could this be changed to `maximum_convective_updraft_mass_flux` (here and in GFS_typedefs.meta)?

Reply:
@grantfirl I will revise it accordingly. Thanks.